tdrs-backend/tdpservice/settings/common.py

```diff
@@ -452,15 +452,15 @@ class Common(Configuration):
     CELERY_TIMEZONE = 'UTC'
 
     CELERY_BEAT_SCHEDULE = {
-        'name': {
+        'Database Backup': {
             'task': 'tdpservice.scheduling.tasks.postgres_backup',
             'schedule': crontab(minute='0', hour='4'), # Runs at midnight EST
             'args': "-b",
             'options': {
                 'expires': 15.0,
             },
         },
-        'name': {
+        'Account Deactivation Warning': {
```

Comment:
overriding the `celery_beat_schedule` with the same name was committed November 2022 - https://github.com/raft-tech/TANF-app/pull/2204